chirp/drivers/kguv9dplus.py

```diff
@@ -2445,6 +2449,14 @@ class KGUV9PXRadio(KGUV9DPlusRadio):
     _rev = b"02"  # default rev for the radio I know about...
     _file_ident = b"kg-uv9px"
     NEEDS_COMPAT_SERIAL = False
+    _valid_steps = STEPS
+    _step_list = STEP_LIST
+
+    # Force Match Model to False to prevent Tox error for multiple drivers
```

Comment:
It's not tox. Tox is just running our tests. If we change to using something else (we haven't always used tox, and may not forever) then this won't make any sense :)

Further, the test isn't just there to make life difficult, it's failing because chirp itself wouldn't be able to tell the difference between the two images. Meaning a user might save radio B and then when they re-open the image, it might be detected as radio A.

Jim is pretty good about using a standard comment for all his radios, so maybe it'd be good to just replicate that:

https://github.com/kk7ds/chirp/blob/master/chirp/drivers/btech.py#L4049